patterns/1-initial/not-invented-here.md

```diff
@@ -0,0 +1,136 @@
+## Title
+
+Overcoming the Not-Invented-Here Mindset
+
+## Patlet
+
+TBD
```

Comment:
```suggestion
Perfectly good solutions are being rejected because of "Not Invented Here" (NIH). Engineers and their managers will choose to implement functionality themselves first, even if an alternative exists. A shift towards a culture of "Proudly Found Elsewhere" can help reduce the negative impact of NIH.
```

Add draft Patlet based on information in the PR and a bit from the Solution section.